src/app/components/about/about.component.html

```diff
@@ -1,31 +1,28 @@
-					<a href="http://cloudcv.org" target="_blank" class="blue-text">CloudCV</a>. CloudCV aims to make AI research 
-					reproducible and easily accessible. We want to reduce the barrier to entry for doing research 
-					and make it easier for researchers, students and developers to develop and use state-of-the-art 
-					algorithms as a service.</p>
-				<p class="about-content">With EvalAI, we want to standardize the process of 
-					evaluating different methods on a dataset and make it simple to host a competition.
-					Comparing a new method with other existing approaches is an essential component of 
-					research and this process has traditionally been affected by difference in evaluation 
-					metric implementation, different splits of the dataset or minor modifications in the
-					algorithm. This makes it extremely hard to reproduce numbers from published papers 
-					and reliably compare your method with other existing approaches. By building EvalAI, we 
-					hope to make this easier by standardizing the dataset splits, evaluation metrics and by 
-					maintaining a public leaderboard of hosted challenges.</p>
-			</div>
-			<div class="col-lg-5 col-md-5 col-sm-1 col-xs-1">
-				<div class="rocket-container">
-					<div class="spaceman">
-						<img src="assets/images/spaceman.png">
-					</div>
+<div class="about-container">
+	<div class="row">
+		<div class="col-lg-7 col-md-7 col-sm-12 col-xs-12 about-content-column">
+			<h4 class="about-title">About Us</h4>
+			<p class="about-content">EvalAI is built by a team of open source enthusiasts working at 
+				<a href="http://cloudcv.org" class="blue-text">CloudCV</a>. CloudCV aims to make AI research 
+				reproducible and easily accessible. We want to reduce the barrier to entry for doing research 
+				and make it easier for researchers, students and developers to develop and use state-of-the-art 
+				algorithms as a service.</p>
+			<p class="about-content">With EvalAI, we want to standardize the process of 
+				evaluating different methods on a dataset and make it simple to host a competition.
+				Comparing a new method with other existing approaches is an essential component of 
+				research and this process has traditionally been affected by difference in evaluation 
+				metric implementation, different splits of the dataset or minor modifications in the
+				algorithm. This makes it extremely hard to reproduce numbers from published papers 
+				and reliably compare your method with other existing approaches. By building EvalAI, we 
+				hope to make this easier by standardizing the dataset splits, evaluation metrics and by 
+				maintaining a public leaderboard of hosted challenges.</p>
+		</div>
+		<div class="col-lg-5 col-md-5 col-sm-1 col-xs-1">
+			<div class="rocket-container">
+				<div class="spaceman">
+					<img src="assets/images/spaceman.png">
 				</div>
 			</div>
 		</div>
-    </div>
-<app-footer></app-footer>
+	</div>
```

Comment:
Please format the file

Reply:
The file is already formatted. See the screenshot below. I guess what you are seeing is the second to the last line.

![Screenshot (123)](https://user-images.githubusercontent.com/35180217/70569792-e6d73b00-1b9a-11ea-8654-690c863fa5c5.png)

Reply:
The file is already formatted. See the screenshot below. I guess what you are seeing is the second to the last line.

![Screenshot (124)](https://user-images.githubusercontent.com/35180217/70570290-e55a4280-1b9b-11ea-9200-220edb1b8ee9.png)